docs/ContributionGuide.md

```diff
@@ -1,16 +1,17 @@
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+**Table of Contents**  
```

Comment:
I could not get the benefit of changing the existing development guides 🤔 Looks harder to maintain than before (now you need to specify like this is only for extension module)

Reply:
yeah, you are correct. But at this moment, if someone wants to make PR, all the information to setup the project locally is there. I think at some point we need to separate the files, but by looking at them right now, one contribution guide for the repository makes sense for me.